app/workers/fast_track/hypertension_pdf_generator.rb

```diff
@@ -0,0 +1,242 @@
+# frozen_string_literal: true
+
+module FastTrack
+  class HypertensionPdfGenerator
+    attr_accessor :patient, :bp_data, :medications
+
+    def initialize(patient, bp_data, medications, date)
+      @patient = patient
+      @bp_data = bp_data
+      @medications = medications
+      @date = date
+    end
+
+    def generate
+      pdf = Prawn::Document.new
+      pdf = add_intro(pdf)
+      pdf = add_blood_pressure(pdf)
+      pdf = add_medications(pdf) if medications.length > 1
+      add_about(pdf)
+    end
+
+    def stringify_patient
+      suffix = patient[:suffix].present? ? ", #{patient[:suffix]}" : ''
+      stringified = ''
+      %i[first middle last].each do |name|
+        stringified = "#{stringified} #{patient[name]}" if patient[name].present?
```

Comment:
Should this be `+=` instead of `=`?

Reply:
It could be rewritten as `stringified += " #{patient[name]}" if patient[name].present?` but at the time I thought the explicit version was clearer.

Reply:
OHHH. Well, in this instance, it wasn't. The `+=` syntax is what I was expecting. I didn't even see the call to stringified instead the string.

Here's another alternative to this whole method:
```ruby
full_name = [patient[:first], patient[:middle], patient[:last]].compact.join ' '
[full_name, patient[:suffix]].compact.join ', '
```

Thoughts?

If you expect their name to be an empty string (rather than nil), you can use `reject(&:blank?)` instead of `compact`.

Reply:
Thanks, I adopted your suggestion in https://github.com/department-of-veterans-affairs/vets-api/commit/dcfd5b5f118ff4feabc204056a6496f4962457a1.